modules/manually-creating-alibaba-ram-user.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/manually-creating-alibaba-ram.adoc
+
+:_content-type: PROCEDURE
+[id="manually-creating-alibaba-ram-user_{context}"]
+= Creating the required RAM user
+
+// https://github.com/openshift/cloud-credential-operator/pull/412/files#diff-2480a11ca4927139d6eaa9883946b6f4cb38358cd98def8c57dd73e9319dbc9cR232
+
+You must create a Alibaba Cloud Resource Access Management (RAM) user for the installation by using the RAM console. Later, you create credentials in {product-title} based on this user's permissions.
+
+When you configure the RAM user, be sure to consider the following requirements:
+
+* Select `Open API Access` for the Access Mode when creating the user. This mode generates the required Alibaba Cloud AccessKeyID and AccessKeySecret.
```

Comment:
@kwoodson @mtulio @joelddiaz Is this correct? When creating a RAM user, there are two options for Access Mode. This one creates the AK ID and Secret.

Reply:
@mburke5678 LGTM. It's required when creating, but it's also possible to create when the user already exists.